game.py

```diff
@@ -570,6 +582,7 @@ def UnidentifiedInput(self):
                 print("put take smokes")
             elif self.Place == "Gryphon":
                 print("do ou wanna fight him or not??")
+
 if __name__ == "__main__": 
-  starts = Player(100, 0, 0, False, False, False, False)
-  starts.CheckpointLoad()
\ No newline at end of file
+  starts = Player()
+  starts.CheckpointLoad()

```

Comment:
does this even work?

Reply:
Yes, I switched it to default arguments specified in `__init__` you might wanna make `Player.CheckPointLoad` a class method though